### ISSUE TITLE
Listing Grid sort by views

### ISSUE THREAD

**Issue by TheBigBanik** [CrocoBlock/suggestions#1692]
Hi, although listing grid has many features, it lucks the possibility to sort by most viewed posts. This would be a great addition and a must if we need to build more flexible and useful websites for our customers.

**Comment by andretoldo** [comment]
I'm trying to find an alternative for this issue but with no success at the moment. Please Crocoblock team, this would be a great update =)

**Comment by medialoods** [comment]
This would be an awesome feature.

**Comment by tauseefch** [comment]
Yes I am also looking for this feature it will be great feature...!

**Comment by aalexandr21** [comment]
You can create a data store https://crocoblock.com/knowledge-base/jetengine/how-to-create-recently-viewed-section-using-data-stores-module/

Then create a Post Query in the Query Builder with next set-up https://crocoblock.com/knowledge-base/tips-and-tricks/cannot-output-the-most-popular-posts-via-listing-grid/
